src/api/models/iuf/manifestDataValidation/manifest_data_validate.go

```diff
@@ -108,6 +111,38 @@ func (vs *validators) validateNexusRepoFilePath() error {
 	return nil
 }
 
+// Method to process nexus repo content, returns repo file path and error(in case of issues),
+func (vs *validators) processNexusRepoSchemaFile(nexusContentRaw interface{}) error {
+
+	logger := utils.GetLogger()
+
+	// Nexus repo possible schemas
+	vs.nexusSchemaFileNames = append(vs.nexusSchemaFileNames, "schemas/nr-hosted-repo-schema.yaml")
```

Comment:
Lets have a logic which will return all yaml files in schemas folder, so we can remove all hard coding, in case of new schema files , code will be immune.